modules/test/playwright/tests/export-import-web/import.spec.ts

```diff
@@ -9,13 +9,13 @@ import {expect, mergeTests} from '@playwright/test';
 import * as path from 'path';
 
 import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
-import {documentLibraryPagesTest} from '../../fixtures/documentLibraryPages.fixtures';
+import {documentLibraryPages} from '../../fixtures/documentLibraryPages';
```

Comment:
I'm not sure if this rename is needed but I'm not sure about the current conventions.

Reply:
I just saw that the other fixtures don't include the "test" at the end of the name, and makes more sense to me. not any special reason